docs/metrics.md

```diff
@@ -1,134 +1,84 @@
-# Exposed Metrics
+# KubeVirt metrics
 
-Sometimes the Help text on `/metrics` endpoint just isn't enough to explain what a certain metric means. This document's objective is to give further explanation of KubeVirt related metrics.
+This document aims to help developers that are not familiar with all metrics exposed by different KubeVirt components. 
+All metrics documented here are auto-generated by the utility tool`tools/doc-generator` and reflects exactly what is being exposed.
 
-## Kubevirt Metric
+## Kubevirt specific metrics
 
 #### kubevirt_info
+#### HELP Version information. 
 
-Kubevirt's version information
+## VMI metrics
+#### kubevirt_vmi_phase_count
+#### HELP kubevirt_vmi_phase_count sum of VMIs per phase and node. `phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`]
 
-Labels:
-* `goversion` - GO version used to compile this version of KubeVirt 
-* `kubeversion` - Git commit refspec that created this version of KubeVirt 
 
-## Node Metric
+#### kubevirt_vmi_vcpu_seconds
+#### HELP kubevirt_vmi_vcpu_seconds amount of time spent in each state by each vcpue. Where `id` is the vcpu identifier and `state` can be one of the following: [`OFFLINE`, `RUNNING`, `BLOCKED`]
```

Comment:
Maybe we can give the `HELP` only three `#` instead of 4 and strip away the `HELP`?

Reply:
Just writing it as normal test may also make sense.

Reply:
I've stripped away the `HELP` and the metric name that follows it, WDYT?